mkat_fpga_tests/test_cbf.py

```diff
@@ -1470,3 +1470,51 @@ def test_all_delays(self):
             Aqf.less(abs_diff, 1,'Check that the maximum degree between '
                 'expected and actual phase difference between intergrations '
                 'is below 1 degree: {0:.3f} degree\n'.format(abs_diff))
+    @aqf_vr('TP.C.1.17')
+    def test_config_report(self):
+        """CBF Report configuration"""
+
+        repos = ['corr2', 'mkat_fpga', 'casperfpga', 'mkat_fpga_tests',
```

Comment:
Is it viable to import corr2 and casperfpga modules only to check where their running from?